src/components/shared/Extensions/Accordion/partials/ConnectForm.tsx

```diff
@@ -0,0 +1,13 @@
+import { useFormContext } from 'react-hook-form';
+
+const displayName = 'shared.Extensions.Accordion.partials.ConnectForm';
```

Comment:
`const displayName = 'Extensions.Accordion.partials.ConnectForm';`

As I can see, components in the `shared` directory don't require a `shared` prefix in displayName.